py/server/deephaven_internal/jvm/__init__.py

```diff
@@ -17,12 +19,29 @@ def check_ready():
         raise Exception("The Deephaven Server has not been initialized. "
                         "Please ensure that deephaven_server.Server has been constructed before importing deephaven.")
 
+
+def check_py_env():
+    """Checks if the current Python environment is in good order."""
+    import importlib.metadata
+    try:
+        importlib.metadata.version("deephaven")
+    except:
+        pass
+    else:
```

Comment:
I didn't know about this `try / except / else` construct (specifically the `else` part) - cool.